integration-tests/pom.xml

```diff
@@ -374,21 +375,23 @@
                         <artifactId>exec-maven-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>build-and-start-druid-cluster</id>
+                                <id>docker-package</id>
                                 <goals>
                                     <goal>exec</goal>
                                 </goals>
                                 <phase>pre-integration-test</phase>
                                 <configuration>
                                     <environmentVariables>
-                                    <DRUID_INTEGRATION_TEST_START_HADOOP_DOCKER>${start.hadoop.docker}</DRUID_INTEGRATION_TEST_START_HADOOP_DOCKER>
-                                    <DRUID_INTEGRATION_TEST_SKIP_START_DOCKER>${skip.start.docker}</DRUID_INTEGRATION_TEST_SKIP_START_DOCKER>
-                                    <DRUID_INTEGRATION_TEST_JVM_RUNTIME>${jvm.runtime}</DRUID_INTEGRATION_TEST_JVM_RUNTIME>
-                                    <DRUID_INTEGRATION_TEST_GROUP>${groups}</DRUID_INTEGRATION_TEST_GROUP>
-                                    <DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH>${override.config.path}</DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH>
-                                    <DRUID_INTEGRATION_TEST_RESOURCE_FILE_DIR_PATH>${resource.file.dir.path}</DRUID_INTEGRATION_TEST_RESOURCE_FILE_DIR_PATH>>
+                                        <DRUID_INTEGRATION_TEST_START_HADOOP_DOCKER>${start.hadoop.docker}</DRUID_INTEGRATION_TEST_START_HADOOP_DOCKER>
+                                        <DRUID_INTEGRATION_TEST_JVM_RUNTIME>${jvm.runtime}</DRUID_INTEGRATION_TEST_JVM_RUNTIME>
+                                        <DRUID_INTEGRATION_TEST_GROUP>${groups}</DRUID_INTEGRATION_TEST_GROUP>
+                                        <DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH>${override.config.path}</DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH>
+                                        <DRUID_INTEGRATION_TEST_RESOURCE_FILE_DIR_PATH>${resource.file.dir.path}</DRUID_INTEGRATION_TEST_RESOURCE_FILE_DIR_PATH>
+                                        <DRUID_INTEGRATION_TEST_SKIP_BUILD_DOCKER>${skip.build.docker}</DRUID_INTEGRATION_TEST_SKIP_BUILD_DOCKER>
+                                        <DRUID_INTEGRATION_TEST_SKIP_RUN_DOCKER>${skip.start.docker}</DRUID_INTEGRATION_TEST_SKIP_RUN_DOCKER>
```

Comment:
I keep "start" because it was added before.
Actually I dont like all variables: "skip.*.docker" because they have wrong naming. It is better to name it: "docker.build.skip", "docker.run.skip", "docker.stop.skip"
And I think that we dont need "docker.stop.skip" variable because if you dont want to "run" cluster then we need to skip step "stop".

Reply:
The purpose of DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER is only to prevent the tearing down of Druid cluster after the test completed. This is so that you can continue to inspect the cluster after the test finished.

Now that I think about it, maybe we can remove DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER. I actually don't think it is that useful. Also, feel free to rename the variables. I also think "docker.build.skip", "docker.run.skip", "docker.stop.skip" are better. Please also update the README if you do rename them.

Reply:
Fixed.
New variable names: "docker.run.skip", "docker.build.skip"
Removed "skip.stop,docker" variable
Updated readme